.env.example

```diff
@@ -0,0 +1,8 @@
+#DATABASE
+DATABASE_URL=
+
+#TWILLIO
+ACCOUNT_SID=
```

Comment:
Twilio is not present in this PR, should be extracted to a separate branch.

Reply:
++